server/issue.go

```diff
@@ -11,7 +11,7 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/andygrunwald/go-jira"
+	jira "github.com/andygrunwald/go-jira"
```

Comment:
10/10 a nit, but why? `go-jira` resolves as `jira`, right? Intention-revealing?

Reply:
@levb gofmt (or go imports)'s choice. Why now, is my question...